.changeset/new-timers-warn.md

```diff
@@ -0,0 +1,11 @@
+---
+'@aws-amplify/integration-tests': minor
+'@aws-amplify/backend-deployer': minor
+'create-amplify': minor
+'@aws-amplify/plugin-types': minor
+'@aws-amplify/cli-core': minor
+'@aws-amplify/sandbox': minor
+'@aws-amplify/backend-cli': minor
+---
+
+Support more package managers, such as yarn (version 1 and version 2+), pnpm.
```

Comment:
Do we also want to add several examples in the changelog?

I'm thinking of adding:

> For example the following commands:
> `yarn create amplify`
> `yarn amplify pipeline-deploy`
> `pnpm create amplify`
> `pnpm amplify pipeline-deploy`

Reply:
yes why not

Reply:
however if you do you may want to split changeset into two files and add this only for CLI and create amplify